test/command_callback/test_gawk_command_callback.vader

```diff
@@ -0,0 +1,27 @@
+Before:
```

Comment:
Looks good. Add a test here to check that you can still set other options for `gawk`.